raspberry/bzzz/sensors/gnss.py

```diff
@@ -169,52 +275,114 @@ def all_gnss_data(self):
         This method returns all sensor data after the application of the data
         preprocessor specified in the constructor. The data is returned as a
         numpy array with the following data (in this order):
+          - Relative North position of quadcopter to base station in meters.
+          - Relative East position of quadcopter to base station in meters.
+          - Relative Down position of quadcopter to base station in meters.
+          - GNSS status 0:bad 1:ok
+          - RTK status 0:no carrier 1:float 2:fix
           - Latitude in decimal 
           - Longitude in decimal 
-          - Altitude
+          - Altitude/Height
+          - Height above mean sea level
         """
         with self.__lock:
             return self.__data_processor.process(self.__values_cache[:, :], 
                                                  cursor=self.__cursor)
 
     @property
-    def latitude(self):
+    def relative_north(self):
         """
-        Returns Latitude position in decimal
+        The distance of the quadcopter in meters in the north direction
+        relative to the base station
         """
         with self.__lock:
             return self.__data_processor.process(self.__values_cache[:, 0], 
                                                  cursor=self.__cursor)
 
     @property
-    def longitude(self):
+    def relative_east(self):
         """
-        Returns Longitude position in decimal
+        The distance of the quadcopter in meters in the east direction relative
+        to the base station
         """
         with self.__lock:
             return self.__data_processor.process(self.__values_cache[:, 1], 
                                                  cursor=self.__cursor)
+        
+    @property
+    def relative_down(self):
+        """
+        The distance of the quadcopter in meters in the down direction relative
+        to the base station
+        """
+        with self.__lock:
+            return self.__data_processor.process(self.__values_cache[:, 2], 
+                                                 cursor=self.__cursor)
 
     @property
     def altitude(self):
```

Comment:
According to the documentation you've written here, both `altitude` and `relative_down` return the same thing. Is this correct?

Reply:
I've updated the documentation and changed one of the variables names, does it make sense now @alphaville

Reply:
Yes, thanks a lot!